tests/modeling/test_models.py

```diff
@@ -486,83 +472,79 @@ def test_deployment_update_step_order(self):
+            (True, {}, now, now, {}, None, ServiceModification.STARTED),
         ]
     )
-    def test_deployment_modification_model_creation(
-            self, service_instance_storage, is_valid, context, created_at, ended_at,
-            modified_node_templates, nodes, status):
-        deployment_modification = _test_model(
+    def test_service_modification_model_creation(self, service_storage, is_valid, context,
+                                                 created_at, ended_at, modified_node_templates,
+                                                 nodes, status):
+        service_modification = _test_model(
             is_valid=is_valid,
-            storage=service_instance_storage,
-            model_cls=ServiceInstanceModification,
+            storage=service_storage,
+            model_cls=ServiceModification,
             model_kwargs=dict(
-                service_instance=service_instance_storage.service_instance.list()[0],
+                service=service_storage.service.list()[0],
                 context=context,
                 created_at=created_at,
                 ended_at=ended_at,
-                modified_nodes=modified_node_templates,
-                node_instances=nodes,
+                modified_node_templates=modified_node_templates,
+                nodes=nodes,
                 status=status,
             ))
         if is_valid:
-            assert deployment_modification.service_instance == \
-                   service_instance_storage.service_instance.list()[0]
+            assert service_modification.service == \
+                   service_storage.service.list()[0]
 
 
 class TestNodeTemplate(object):
     @pytest.mark.parametrize(
-        'is_valid, name, default_instances, max_instances, min_instances, plugins, properties, '
-        'type_name, type_hierarchy',
+        'is_valid, name, default_instances, max_instances, min_instances, plugin_specifications, '
+        'properties',
         [
-            (False, m_cls, 1, 1, 1, [], [], 'type', []),
-            (False, 'name', m_cls, 1, 1, [], [], 'type', []),
-            (False, 'name', 1, m_cls, 1, [], [], 'type', []),
-            (False, 'name', 1, 1, m_cls, [], [], 'type', []),
-            (False, 'name', 1, 1, 1, m_cls, [], 'type', []),
-            (False, 'name', 1, 1, 1, [], [], m_cls, []),
-            (False, 'name', 1, 1, 1, [], [], 'type', m_cls),
-            #
-            (True, 'name', 1, 1, 1, [], [], 'type', []),
-            (True, 'name', 1, 1, 1, None, [], 'type', []),
-            (True, 'name', 1, 1, 1, [], [], 'type', None),
+            (False, m_cls, 1, 1, 1, {}, {}),
+            (False, 'name', m_cls, 1, 1, {}, {}),
+            (False, 'name', 1, m_cls, 1, {}, {}),
+            (False, 'name', 1, 1, m_cls, {}, {}),
+            (False, 'name', 1, 1, 1, m_cls, {}),
+            (False, 'name', 1, 1, 1, None, {}),
+
+            (True, 'name', 1, 1, 1, {}, {}),
```

Comment:
Since moving so much to foreign keys, there are much fewer raw columns to test now.